src/sdk/namespace_fs.js

```diff
@@ -332,16 +351,16 @@ class NamespaceFS {
 
             /** @type {Result[]} */
             const results = [];
+            const versions = [];
```

Comment:
why a new array is needed? can't we push the results array?

Reply:
At line 547, the results array is traversed one by one and related version objects inside version dir are fetched, arranged based on c_time. Once  we have sorted version objects, we will push them into `object_info` at line 577 to 581